lib/mixlib/log/logging.rb

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright:: Copyright (c) 2018 Chef Software, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "logger"
+require "mixlib/log/child"
+
+module Mixlib
+  module Log
+    module Logging
+
+      module Severity
+        include ::Logger::Severity
+        TRACE = -1
+
+        SEV_LABEL = %w{TRACE DEBUG INFO WARN ERROR FATAL ANY}.each(&:freeze).freeze
+
+        def to_label(sev)
+          SEV_LABEL[sev + 1] || -"ANY"
+        end
+      end
+      include Severity
```

Comment:
defining the module only to include it seems a bit unusual here...  if there's a reason, could use a comment...